target/linux/ipq806x/base-files/lib/upgrade/platform.sh

```diff
@@ -19,13 +29,9 @@ platform_do_upgrade() {
 		CI_ROOTPART="ubi_rootfs"
 		nand_do_upgrade "$1"
 		;;
-	compex,wpq864|\
-	netgear,d7800 |\
-	netgear,r7500 |\
-	netgear,r7500v2 |\
-	netgear,r7800 |\
-	qcom,ipq8064-ap148 |\
-	qcom,ipq8064-ap161)
+	buffalo,wxr-2533dhp)
+		buffalo_upgrade_prepare_ubi
+		CI_ROOTPART="ubi_rootfs"
 		nand_do_upgrade "$1"
```

Comment:
A wild buffalo appears out of nowhere!

Reply:
Note to self: never trust that git rebase will do everything correctly